Apply review feedback

meker12 opened the pull request:
- Update intro topic per feedback from Rob Jackson
- Change product name variable from Brocade ADX Brocade ADX product name in the conf.py
  variable
- Edits and format fixes

Commit:
Apply review feedback

- Update intro topic per feedback from Rob Jackson
- Change product name variable from Brocade ADX Brocade ADX product name in the conf.py
  variable
- Edits and format fixes

rackernexus commented:
Your content preview [is now ready](https://staging.developer.rackspace.com/build-cb84d83b28/docs/dedicated-load-balancers/v2/). :bow:

rackernexus commented:
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-b68632be3b/docs/dedicated-load-balancers/v2/).

Commit:
Apply additional edits
- Fix Brocade ADX request headers
- Fix product name references
- Add information about limits and concurrent connection configuration

rackernexus commented:
Your content preview [is now ready](https://staging.developer.rackspace.com/build-a7a45e61e2/docs/dedicated-load-balancers/v2/). :bow:

Commit:
Fix typo.

rackernexus commented:
HULK [PREVIEW](https://staging.developer.rackspace.com/build-18a9410177/docs/dedicated-load-balancers/v2/)

Commit:
Fix API reference title

rackernexus commented:
:mag_right: [Preview](https://staging.developer.rackspace.com/build-bb9bfdee49/docs/dedicated-load-balancers/v2/)
